app/workers/notification_workers/emotion_selection_notification_worker.rb

```diff
@@ -0,0 +1,17 @@
+module NotificationWorkers
+  class EmotionSelectionNotificationWorker
```

Comment:
Repeating "NotificationWorkers" makes for a very long class name, without that space adding real information.  A couple questions:
 * Is putting this in a module necessary? If not, maybe just leave it as a class, e.g. `EmotionSelectionNotificationWorker`.
 * If you have to keep it in a module, maybe this? `NotifiactionWorkers::EmotionSelectionInvite`

Reply:
@rogergraves Tanks. Ok, I did it just to have more namespace in the future.